app/containers/Buttons/RefreshButton/RefreshButton.jsx

```diff
@@ -0,0 +1,39 @@
+// @flow
+import React, { Component } from 'react'
+import classNames from 'classnames'
+
+import RefreshIcon from '../../../assets/icons/refresh.svg'
+
+import styles from './RefreshButton.scss'
+
+type Props = {
+  loadWalletData: () => void,
+  loading?: boolean
+}
+
+class RefreshButton extends Component<Props> {
+  render() {
+    const { loadWalletData, loading } = this.props
+
+    return (
+      <div className={styles.refreshButton}>
+        <span
+          onClick={loading ? null : loadWalletData}
+          className={styles.refreshButtonSpan}
+        >
+          {' '}
```

Comment:
Pretty sure we do not need these empty strings

Reply:
they often get added by style formatters automatically

Reply:
@dvdschwrtz is right. This is Prettier's work.